psalm.xml

```diff
@@ -20,6 +20,10 @@
         <directory name="lib/SimpleSAML/Utils" />
         <directory name="lib/SimpleSAML/XHTML" />
         <directory name="lib/SimpleSAML/XML" />
+
+        <!-- Replaces all modules/... with this one-liner for 2.0
+        <directory name="modules" />
+        -->
```

Comment:
We need this per-module as long as we're requiring all externalized modules in the build